packages/firebase_ui_localizations/lib/src/all_languages.dart

```diff
@@ -3,6 +3,8 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'dart:ui';
 
+import 'package:firebase_ui_localizations/src/lang/nb.dart';
```

Comment:
please follow the same style for imports as the rest of the languages

Reply:
Done 👍